src/codeview.jl

```diff
@@ -1,12 +1,20 @@
 const asm_syntax = Ref(:att)
 
+function highlight(io, lexer, x)
+    Sys.which("pygmentize") === nothing && return print(io, x)
```

Comment:
This checks should be outline to a build step

Reply:
Is outlining necessary for interactive tools like Cthulhu?  I understand that this is important when the existence and details of the external tools matter in the performance sensitive part (like CUDA).  But `Sys.which` is negligible compared to human interactions.  On the other hand, being able to use code highlighting right after installing `pygmentize` without needing to re-build/precompile seems to be a slight win.

Reply:
I like efficiency ;), but as long as I can turn it off, when it breaks. Preferably it should be a toggle like optimized

Reply:
I do like efficiency too.  My point is that it is highly inefficient to require users to restart entire Julia REPL to enable or disable highlighting.  (OK, if they use Revise.jl maybe that's fine.  But basing the interface design on to Revise.jl does not seem like a good idea to me.)

For the same reason, I don't think hiding it behind the toggling UI is a good idea.  The biggest bottleneck here is the human keystroke.  Saving single keystroke at the cost of single `Sys.which` is a huge win.

Reply:
By putting it behind `Sys.which` you are forcing me to remove `pygments` to disable it...

1. Precompute during `__init__` and put the result into a global `Ref`.
2. There needs to be a toggle to turn it *off* at least, I don't want to restart a `descend` just because the highlighter broke

Reply:
That's also the reason why I was suggesting to add a global configuration.  It didn't occurred to me that you'd want to toggle highlighter on-the-fly but it makes sense, too.

So how about something like this?

```julia
Base.@kwdef mutable struct CthulhuConfig
    enable_highlighter::Bool = false
    highlighter::Cmd = `pygmentize`
end

const CONFIG = CthulhuConfig()

function __init__()
    CONFIG.enable_highlighter = Sys.which(CONFIG.highlighter) !== nothing
end

function _descend(...)
    ...
        elseif toggle === :highlighter
            # Make the UI toggle "sticky"; i.e., the next descent will remember
            # the setting.
            CONFIG.enable_highlighter ⊻= true
    ...
end
```

The reason why I separated `enable_highlighter` and `highlighter` is that it lets you toggle highlighter without discarding user setting in `highlighter`.

This is similar to how ElectronDisplay.jl handles configuration: https://github.com/queryverse/ElectronDisplay.jl/blob/4d0c799017d1f1cfc7ee42d82040b3666e7b7f66/src/ElectronDisplay.jl#L43-L61

Reply:
That looks good!